docs/docs.md

```diff
@@ -991,20 +1003,20 @@ A single random integer in the given inclusive range.
 @(rand_between(1, 10)) → 10
 ```
 
-<a name="function:read_code"></a>
+<a name="function:read_digits"></a>
 
-## read_code(code)
+## read_digits(code)
```

Comment:
Old function name here `read_digits` seems clearer

Reply:
I think the change was because it isn't number specific.. IE, could be 6 alphanums, but you want them spaced out.

Reply:
Ah ok... I can revert or can we think of something more specific than "code"... "chars" ?